### PR TITLE
clang-format

### PR DESCRIPTION
**Pull request by RocioRojas**
### Contribution description
change SortIncludes in clang format
*SortIncludes: false

### Testing procedure
Look at clang format line 110

### Issues/PRs references
none

### CODE REVIEW

**Comment by luisan00** [comment]
AT commads for the shell should be more human readable, for example
`won` -> `wifi_on`
`woff` -> `wifi_off`
Also keep in mind some commands will set a value (=) but others may query for a value or status (?)

File: examples/at_uart/main.c [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+ * @author  eduazocar <eduardo@turpialdev.com>
+ */
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "board.h"
+#include "msg.h"
+#include "shell.h"
+#include "shell_commands.h"
+#include "thread.h"
```
Comment (luisan00) [inline examples/at_uart/main.c:30]: why this?

File: examples/at_uart/main.c [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+#include "xtimer.h"
+
+#define AT_UART UART_DEV(1)
+#define AT_BAUDRATE 115200
+#define MAX_SENTENCE_SIZE 128
+
+#define MAIN_QUEUE_SIZE (8)
+msg_t _main_msg_queue[MAIN_QUEUE_SIZE];
+
+typedef struct t_at_list {
```
Comment (luisan00) [inline examples/at_uart/main.c:41]: this name is not aligned with our CC

File: examples/at_uart/main.c [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+    char *shell_cmd;
+    char *value_desc;
+    char *at_cmd;
+    uint8_t arg;
+} tAT_LIST;
```
Comment (luisan00) [inline examples/at_uart/main.c:46]: and this too

File: examples/at_uart/main.c [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+
+enum AT_COUNT {
```
Comment (luisan00) [inline examples/at_uart/main.c:48]: name unaligned with our CC

File: examples/at_uart/main.c [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+    printf("%s\n", (char *)arg);
+    if (c != 10) {
+        uart_buff[count] = (char)c;
+        count += 1;
+    } else {
+        printf("%s\n", (char *)uart_buff);
+        memset(uart_buff, 0, sizeof(uart_buff));
+        count = 0;
+    }
+}
```
Comment (luisan00) [inline examples/at_uart/main.c:92]: this is taken from an older example, remove it or adjust it for your requirements

File: examples/at_uart/main.c [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+#include "msg.h"
+#include "shell.h"
+#include "shell_commands.h"
+#include "thread.h"
+#include "periph/uart.h"
+#include "xtimer.h"
+
+#define AT_UART UART_DEV(1)
+#define AT_BAUDRATE 115200
+#define MAX_SENTENCE_SIZE 128
```
Comment (luisan00) [inline examples/at_uart/main.c:36]: this must be adjusted or deleted

File: examples/at_uart/main.c [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+    {"nvs_rst", "", "AT&NVS_RST", 0},
+    {"wmode", "<0 / 1 / 2 / 3>", "AT+WIFI_MODE=", 1},
+    {"wap_chan", "<Number>", "AT+WAP_CHAN=", 1},
+    {"wap_auth", "<Number as AUTH_TYPE>", "AT+WAP_AUTH=", 1},
+    {"wrst", "", "AT&WIFI_RST\r\n", 0},
+    {NULL, NULL, NULL, 0},
+};
+
+size_t count = 0;
+char uart_buff[MAX_SENTENCE_SIZE] = {0};
```
Comment (luisan00) [inline examples/at_uart/main.c:80]: change it according your requirements or delete it if not used

File: examples/at_uart/main.c [hunk 8]
```diff
@@ -0,0 +1,166 @@
...
+            }
+        }
+        if (at_list[at_position].arg == 0) {
+            return -1;
+        }
+        sprintf(buff_at, "%s%s\r", at_list[at_position].at_cmd, ssid_buf); // Review this//
+        printf("%s", buff_at);
+        uart_poweron(AT_UART);
+        uart_write(AT_UART, (uint8_t *)buff_at, strlen(buff_at) - 1);
+        uart_poweroff(AT_UART);
```
Comment (luisan00) [inline examples/at_uart/main.c:151]: this may cause some unwanted behaviour

File: examples/at_uart/main.c [hunk 9]
```diff
@@ -0,0 +1,166 @@
...
+    if (argc == 2) {
+
+        if (at_list[at_position].arg == 1) {
+            return -1;
+        }
+        sprintf(buff_at, "%s", at_list[at_position].at_cmd);
+        printf("%s", buff_at);
+        uart_poweron(AT_UART);
+        uart_write(AT_UART, (uint8_t *)buff_at, strlen(buff_at) - 1);
+        uart_poweroff(AT_UART);
```
Comment (luisan00) [inline examples/at_uart/main.c:131]: this may cause some unwanted behaviour

**Review by luisan00 — CHANGES_REQUESTED** [review]
starting with rounds of reviews xD

File: .clang-format [hunk 10]
```diff
@@ -107,7 +107,7 @@ PenaltyExcessCharacter: 1000000
 PenaltyReturnTypeOnItsOwnLine: 60
 PointerAlignment: Right
 ReflowComments:  true
-SortIncludes:    true
+SortIncludes:    false
```
Comment (luisan00) [inline .clang-format:110]: this should be into another PR, this is: 
> example: at_uart #155

File: firmware/libs/at_commands_uart/Makefile [hunk 11]
```diff
@@ -0,0 +1,32 @@
+# name of your application
+APPLICATION = at_uart
```
Comment (luisan00) [inline firmware/libs/at_commands_uart/Makefile:2]: 1.- Wrong makefile
2.- this should be into another PR, this PR name is: 
> example: at_uart #155

**Review by luisan00 — CHANGES_REQUESTED** [review]
another round xD

File: examples/at_uart/Makefile [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+QUIET ?= 1
+
+# Disable warnings when a parameter is not used
+# CFLAGS += -Wno-unused-parameter
+# CFLAGS += -Wno-unused-variable
+
+# Set log level
+CFLAGS += -DLOG_LEVEL=LOG_DEBUG
+
+include $(RIOTBASE)/Makefile.include
```
Comment (eduazocar) [inline examples/at_uart/Makefile:32]: add EOF

**Review by eduazocar — APPROVED** [review]
Well this look good for me :+1:

**Pull request closed without merging**